distributed/scheduler.py

```diff
@@ -657,134 +778,277 @@ class TaskState:
     :      The group of tasks to which this one belongs.
     """
 
+    _key: str
+    _hash: Py_hash_t
+    _prefix: object
+    _run_spec: object
```

Comment:
bytes maybe

Reply:
Interesting. This came from the [docstring above]( https://github.com/dask/distributed/blob/dcdb4651eb0331bb99caf3362c84d4878d94ba5b/distributed/scheduler.py#L438 ). Later in the code it was implied this [could be a `dict`, but might not be]( https://github.com/dask/distributed/blob/dcdb4651eb0331bb99caf3362c84d4878d94ba5b/distributed/scheduler.py#L2822-L2823 ). Seems like it is not very clear what it may be so perhaps it is best to leave as `object` for now.